spec/support/protos/map-test.pb.rb

```diff
@@ -12,8 +12,8 @@ module Foo
   # Enum Classes
   #
   class Frobnitz < ::Protobuf::Enum
-    define :FROB, 1
-    define :NITZ, 2
+    define :FROB, 0
```

Comment:
The first enum must be zero according to protoc. The map-test.proto and map-test.bin have been updated to make the existing tests pass.